content/information-distribution/node.md

```diff
@@ -82,58 +82,20 @@ provide for this room.
-server name as its FQDN, and the default Matrix encrypted federation port,
-`8448`, as its port.
-
-Most Matrix homeservers can also be reached through the standard `443` HTTPS
-port. Whether there's a Matrix `SRV` DNS record for a node or not, client
-implementations **should** try reaching the said node using the `443` port
-before trying reaching it using any other port. This means a node's address and
-port resolution **should** follow the following workflow:
-
-1. Try retrieving a Matrix `SRV` DNS record from the node's server name.
-2. If the DNS record exists, try reaching the node using the FQDN extracted from
-the record and the `443` port.
-3. If step 2 succeeds, consider the FQDN extracted from the DNS record and the
-`443` port as the final parameters to reach the node (i.e. consider the node
-unreachable if those parameters don't work).
-4. If step 2 fails, consider the FQDN and port extracted from the DNS record as
-the final parameters to reach the node (i.e. consider the node unreachable if
-those parameters don't work).
-5. If the DNS record doesn't exist, try reaching the node using its server name
-as its FQDN and the `443` port.
-6. If step 5 succeeds, consider the node's server name and the `443` port as the
-final parameters to reach the node (i.e. consider the node unreachable if those
-parameters don't work).
-7. If step 5 fails, consider the node's server name and the `8448` port as the
-final parameters to reach the node (i.e. consider the node unreachable if those
-parameters don't work).
-
-This should result in making the network traffic generated by Informo client
-implementations more difficult to detect and filter on.
-
-If a client implementation's maintainer(s) decide not to use port 443, their
-client implementation **must** implement the following workflow:
-
-1. Try retrieving a Matrix `SRV` DNS record from the node's server name.
-2. If the DNS record exists, consider the FQDN and port extracted from the
-DNS record as the final parameters to reach the node (i.e. consider the node
-unreachable if those parameters don't work).
-3. If the DNS record doesn't exist, consider the node's server name and the
-`8448` port as the final parameters to reach the node (i.e. consider the node
-unreachable if those parameters don't work).
-
-{{% notice tip %}}
-Web-based client implementations **can** use the [DNS over
-HTTPS](https://tools.ietf.org/html/rfc8484) standard to perform DNS queries over
-HTTPS instead of standard DNS queries which they might not be able to perform.
+implementations **must** implement the server discovery through `.well-known`
+URI logic [described in the Matrix
+specifications](https://matrix.org/docs/spec/client_server/r0.4.0.html#server-discovery).
+
+{{% notice note %}}
+If the server discovery leads to a final `IGNORE` instruction (as specified in
+the link above), then client implementations **must** use the node's server name
```

Comment:
Eventually:
```suggestion
the link above, for example when the `.well-known` file is not found), then client implementations **must** use the node's server name
```

Reply:
I think the section from the Matrix spec is clear enough on how `IGNORE` can be reached, this addition doesn't seem to add much worth, and would need to be maintained if the resolution algorithm change in the future, so I'm not sure about including it.

Reply:
ok, let's follow upstream then :)